lib/active_stash/matchers.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "./assess"
+
+if defined?(RSpec)
+  RSpec::Matchers.define :encrypt_sensitive_fields do
+    match do |model|
+      unprotected(model).empty?
+    end
+
+    failure_message do |model|
+      unprotected = unprotected(model)
+
+      field_text =
+        if unprotected.size > 1
+          "fields"
+        else
+          "field"
+        end
```

Comment:
Rails has the `pluralize` helper for this. But I appreciate you might be avoiding depending on rails too much.

Reply:
Nice, may as well use the helper from Rails since we're explicitly only supporting Rails apps at the moment for Assess.

Reply:
On second thought, I think I'll leave this as it is. We still need to have the condition either way and we only need to make one word plural at the moment.